Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedFigure/GetRelatedFiguresByStreetcodeIdTests.cs

```diff
@@ -0,0 +1,207 @@
+﻿using Moq;
+using AutoMapper;
+using Xunit;
+using Streetcode.BLL.MediatR.Streetcode.RelatedFigure.GetByStreetcodeId;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using Streetcode.BLL.DTO.Streetcode;
+using Microsoft.EntityFrameworkCore.Query;
+using System.Linq.Expressions;
+
+using Entities = Streetcode.DAL.Entities.Streetcode;
+using Streetcode.DAL.Entities.AdditionalContent;
+using Streetcode.DAL.Entities.Streetcode.Types;
+using Streetcode.DAL.Entities.Streetcode;
+using Streetcode.DAL.Entities.Media.Images;
+
+namespace Streetcode.XUnitTest.MediatRTests.Streetcode.RelatedFigure
+{
+
+    public class GetRelatedFiguresByStreetcodeIdTests
+    {
+        [Theory]
+        [InlineData(1)]
+        public async Task GetRelatedFigureByStreetcodeId_Existing(int id)
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testRelatedList = new List<Entities.RelatedFigure>()
+            {
+                    new Entities.RelatedFigure()
+                    {
+                        ObserverId = id,
+                        TargetId = id + 1,
+                        Observer = new StreetcodeContent(){},
+                        Target = new StreetcodeContent(){}
+                    }
+            };
+
+            var testPersonStreetcodeList = new List<PersonStreetcode>()
+            {
+                new PersonStreetcode()
+                {
+                    FirstName= "Test",
+                    LastName = "TestSurname",
+                    Id = id,
+                    Index = id,
+                    Teaser = "test",
+                    EventStartOrPersonBirthDate = DateTime.UtcNow,
+                    EventEndOrPersonDeathDate = DateTime.UtcNow.AddDays(1),
+                }   
+            };
```

Comment:
Why do you need here the whole list if you don't use it?
You can either consider moving this to separate method or using inline data parameters

Reply:
removed all this data, 'cause I am not using it